fedimint-wasm-tests/Cargo.toml

```diff
@@ -3,6 +3,7 @@ name = "fedimint-wasm-tests"
 version = "0.0.0"
 edition = "2021"
 license = "MIT"
+readme = "../README.md"
 publish = false
```

Comment:
Not really needed for `Cargo.toml`s with `publish=false` since they aren't published to crates.io

Reply:
OK , I'll remove them from such Cargo.toml.